data/types/settings/pump/pump.go

```diff
@@ -16,29 +19,50 @@ const (
 	ManufacturersLengthMaximum = 10
 	ModelLengthMaximum         = 100
 	SerialNumberLengthMaximum  = 100
+
+	TimeZoneOffsetMaximum = 7 * 24 * 60 // TODO: Make sure same as all time zone offsets
+	TimeZoneOffsetMinimum = -7 * 24 * 60
+
+	RapidAdult = "rapid_adult"
+	RapidChild = "rapid_child"
```

Comment:
```suggestion
	RapidChild = "rapidChild"
```
`rapid_child` is inconsistent with other strings in this PR.